lms/djangoapps/lti_provider/tests/test_views.py

```diff
@@ -23,6 +23,38 @@
 }
 
 
+COURSE_PARAMS = {
+    'course_id': 'CourseID',
+    'usage_id': 'UsageID'
+}
+
+
+ALL_PARAMS = dict(list(LTI_DEFAULT_PARAMS.items()) + list(COURSE_PARAMS.items()))
```

Comment:
Nitpick: this works, but it's a little difficult to follow the logic, since there are so many transformations. It would be clearer to do:
```python
ALL_PARAMS = {}
ALL_PARAMS.update(LTI_DEFAULT_PARAMS)
ALL_PARAMS.update(COURSE_PARAMS)
```